mica-web-model/src/main/java/org/obiba/mica/web/model/DataAccessRequestDtos.java

```diff
@@ -71,6 +80,20 @@
       builder.addActions("EDIT_STATUS");
     }
 
+    if (!request.isNew()) {
+      try {
+        Project project = projectService.findById(request.getId());
```

Comment:
You get the latest draft version of the project. Certainly not relevant in Drupal.